addons/core/functions/fnc_haveSWRadio.sqf

```diff
@@ -31,6 +31,13 @@ if (isNil "TFAR_currentUnit" || {isNull (TFAR_currentUnit)}) exitWith {false};
     true;
 } count (assignedItems TFAR_currentUnit);
 
+if (player != TFAR_currentUnit) then {
```

Comment:
Okey so then just change the if to
`if (!_result && {player != TFAR_currentUnit})`